pkg/docker/docker.go

```diff
@@ -618,12 +591,12 @@ func updateImageWithInspect(image *api.Image, inspect *dockertypes.ImageInspect)
 
 // RemoveContainer removes a container and its associated volumes.
 func (d *stiDocker) RemoveContainer(id string) error {
+	ctx := context.Background() // no timeout
 	opts := dockertypes.ContainerRemoveOptions{
 		RemoveVolumes: true,
 		Force:         true,
 	}
-	return d.kubeDockerClient.RemoveContainer(id, opts)
-
+	return d.client.ContainerRemove(ctx, id, opts)
```

Comment:
did you disagree with my suggestion on just passing 
```
dockertypes.ContainerRemoveOptions {
   RemoveVolumes: true,
   Force:                  true,
}
```
vs. assigning to `opts` and passing that into `ContainerRemove` ?

Upon revisting, I do see that since `getDefaultContext` returns 2 items you have to do what you did.